thunderbird_labels.php

```diff
@@ -14,6 +14,10 @@ class thunderbird_labels extends rcube_plugin
 	private $rc;
 	private $map;
 	private $_custom_flags_allowed = null;
+	private $message_tb_labels;
+	private $name;
+	private $add_tb_flags;
+	private $list_flags;
```

Comment:
`private $list_flags;` is wrong: it should go in the rcube_message_header class.
this is why I cancelled the PR.

Reply:
oh, so roundcube itself needs patching to completely fix that, I understand